js/src/debugger/Debugger.cpp

```diff
@@ -1423,7 +1423,10 @@ bool Debugger::wrapDebuggeeValue(JSContext* cx, MutableHandleValue vp) {
         name = cx->names().uninitialized;
         break;
       default:
-        MOZ_CRASH("Unsupported magic value escaped to Debugger");
+        mozilla::recordreplay::PrintLog("Unsupported magic value escaped to Debugger %d",
```

Comment:
Do we know why the magic value snuck through in the first place? Would this be because of some replaying issue or something?

Reply:
Oh you commented on the parent issue nevermind.